huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_kafka_smart_connect_test.go

```diff
@@ -0,0 +1,115 @@
+package dms
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getDmsKafkaSmartConnectResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+	// getKafkaSmartConnect: query DMS kafka smart connect
+	var (
+		getKafkaSmartConnectHttpUrl = "v2/{project_id}/instances/{instance_id}"
+		getKafkaSmartConnectProduct = "dms"
+	)
+	getKafkaSmartConnectClient, err := cfg.NewServiceClient(getKafkaSmartConnectProduct, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating DMS Client: %s", err)
+	}
+
+	// Split instance_id from resource id
+	parts := strings.Split(state.Primary.ID, "/")
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("invalid id format, must be <instance_id>/<connector_id>")
```

Comment:
```suggestion
		return nil, fmt.Errorf("invalid id format, must be <instance_id>/<id>")
```
and fix other places

Reply:
done